app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -108,7 +108,9 @@ class AppVersionActivityTest {
     setUpTestApplicationComponent()
     testCoroutineDispatchers.registerIdlingResource()
     val lastUpdateDateTime = context.getLastUpdateTime()
-    lastUpdateDate = getDateTime(lastUpdateDateTime)!!
+    lastUpdateDate = getDateTime(
+      dateTimeTimestamp = lastUpdateDateTime
```

Comment:
This change is not needed.
Revert this to `lastUpdateDate = getDateTime(lastUpdateDateTime)!!`

Reply:
done